main.py

```diff
@@ -1,13 +1,82 @@
-#!/usr/bin/env python
+import json
+import pandas as pd
+from nltk import word_tokenize
+from nltk.corpus import stopwords
+from stop_words import get_stop_words
+from nltk.wsd import lesk
 
-# Your classes or methods go here:
 
+class Writer:
 
-class Tool:
-    def my_method(self):
-        print("my_method called!")
+    """Writer data to json file. Read txt file"""
+
+    @staticmethod
+    def write_to_json(data_text):
+        with open('task.json', 'w') as json_w:
+            json.dump(data_text, json_w, indent=4)
+
+
+class Cleaner:
+
+    @staticmethod
+    def clear_data(table, col_name):
+        table[col_name] = table['original_text'].replace('(\?*[$].+?[ ])', '', regex=True)
+        table[col_name] = table['cleared_text'].replace('https?://(?:[-\w.]|(?:%[\da-fA-F]{2}))+|[@]\w+', '', regex=True)
+        table[col_name] = table['cleared_text'].replace('[#]\w+', '', regex=True)
+        table[col_name] = table['cleared_text'].replace('[^\w\s]', '', regex=True)
+        return table[col_name]
+
+
+class Tokenizer:
+
+    @staticmethod
+    def stop_words(table, col_name):
+        nltk_words = stopwords.words('english')
+        stop_words = get_stop_words('en')
+        stop_words.extend(nltk_words)
+        table[col_name] = table[col_name].apply(lambda without_stopwords: ' '.join(
```

Comment:
This lambda is too crowded with code - could be defined _above_ this line and then passed to the `apply` call. Try to have a single-expression labmdas when you pass them to methods as a param.